app/javascript/App/bundles/NewsfeedPage/utils.ts

```diff
@@ -0,0 +1,43 @@
+import * as _ from 'lodash'
+import { Filters } from './types'
+
+export const SOCIAL_FEED_SOURCES = ['reddit', 'twitter']
+
+export const getDefaultFilters = (): Filters => ({
+  categories: [],
+  coinSlugs: [],
+  feedSources: [],
+  publishedSince: null,
+  publishedUntil: null,
+})
+
+export const TOP_COINS = [
```

Comment:
Hm, the issue with this implementation is that because the crypto markets are so volatile, even the top 5 coins by market cap will change so it's dangerous to hard code.

How difficult would it be to make this a dynamic check instead?

Reply:
Maybe just have a query to grab and cache the top 5 market cap coins daily.

Reply:
If this implementation change will take longer than 2 hours let's just spin up a new ticket for it on the next sprint.

Reply:
To save time I will implement this a root prop and simply pass it via prop drilling into the required components. We can clean this up together with the rest of the NewsfeedPage when we do our refactor.